docs/source/guides/character_control_guide.rst

```diff
@@ -0,0 +1,159 @@
+================================
+NuCypher Character Control Guide
+================================
+
+The `Character Control` is a module that contains useful REST-like HTTP endpoints for working with NuCypher characters.
```

Comment:
I think we need a section (perhaps on index.rst or a separate page) that explains each of the different characters and their roles in the network in clear, plain language. Right now, the docs feel like they're assuming a familiarity with the characters. Maybe this is captures in @arjunhassard's PR.

Reply:
The character control guide is not, imho, meant for explanation of roles. This is strictly for developers who need access to an API. They should familiarize theirselves elsewhere in our documentation.

With that said, I would love to merge @arjunhassard's docs ASAP.